src/entities/Badges/utils.ts

```diff
@@ -112,3 +113,62 @@ export async function getLandOwnerAddresses(): Promise<string[]> {
     return []
   }
 }
+
+const blobToFile = (theBlob: Blob, fileName: string): File => {
+  const b: any = theBlob
```

Comment:
Can you write this like
```
return { ...theBlob, name: fileName, lastModifiedDate: new Date() } as File
```

Reply:
I tried it but it didn't work. Also tried `as any as File` but no luck